packages/help-center/src/components/help-center-content.tsx

```diff
@@ -130,17 +135,10 @@ const HelpCenterContent: React.FC< { isRelative?: boolean; currentRoute?: string
 					<Route
 						path="/"
 						element={
-							navigateToRoute ? (
-								<Navigate to={ navigateToRoute } />
-							) : (
-								<HelpCenterSearch onSearchChange={ setSearchTerm } currentRoute={ currentRoute } />
-							)
```

Comment:
This was intentional to preserve the location state of React Router DOM. The intended way to determine if there is a navigation history that can be navigated to is by checking the `location.key` and verifying if it is not `default`. When the location history is set the key becomes a random string. When we initially navigate on load we are creating a location history and changing the key to be a string despite the fact that the page that was loaded should be considered the initial page.

At the end of the day everything here works so I don't think there is really a problem, but I do think its something that we should do our best to try and preserve for the sake of keeping the `navigate()` option simple and functional. I recently removed specialized location logic that existed within components and complexity within the back button by fixing the location key state. I think moving forward it will prevent technical debt if we can try to maintain the key state.